docs/docs-beta/docs/guides/automation.md

```diff
@@ -93,6 +94,139 @@ Use this table to help guide your decision:
 | Declarative Automation | Asset-centric, condition-based updates | Assets only         |
 | Asset Sensors          | Cross-job/location asset dependencies  | Assets only         |
 
+## GraphQL Endpoint
+
+It is possible to trigger asset materializations in a job from external services using the GraphQL endpoint.
+
+### When to use the GraphQL endpoint
+
+- You want to integrate Dagster with an external system or tool
+- You need to trigger a materialization or job over an HTTP endpoint
+- You are creating a custom script for batching operations
+
+### Triggering a job
+
+To trigger a job to run using the GraphQL endpoint in Dagster, you can use the `launchRun` mutation. Here's an example using the `requests` library:
+
+```python
+import requests
+
+
+graphql_endpoint = "http://localhost:3000/graphql"
+
+query = """
+mutation LaunchRunMutation(
+  $repositoryLocationName: String!
+  $repositoryName: String!
+  $jobName: String!
+  $runConfigData: RunConfigData!
+) {
+  launchRun(
+    executionParams: {
+      selector: {
+        repositoryLocationName: $repositoryLocationName
+        repositoryName: $repositoryName
+        jobName: $jobName
+      }
+      runConfigData: $runConfigData
+    }
+  ) {
+    __typename
+    ... on LaunchRunSuccess {
+      run {
+        runId
+      }
+    }
+    ... on RunConfigValidationInvalid {
+      errors {
+        message
+        reason
+      }
+    }
+    ... on PythonError {
+      message
+    }
+  }
+}
+"""
+
+response = requests.post(
+    graphql_endpoint,
+    json={
+        "query": query,
+        "variables": {
+            "repositoryLocationName": "<replace-with-code-location-name>",
+            "repositoryName": "__repository__",  # default if using `Definitions`
+            "jobName": "<replace-with-job-name>",
+            "runConfigData": {},
+        },
+    },
+)
+```
+
+### Triggering an asset materialization
```

Comment:
Added both asset materialization via `LaunchPipelineExecution` and run request with `LaunchRunMutation`.

How do we feel about the presentation of these two options?